File: python/class 5/question 8.py
weight1_kg = float(input("Enter a weight in kilograms: ")) 
weight1_g = float(input("Enter a weight in grams: "))

weight2_kg = float(input("Enter a weight in kilograms: ")) 
weight2_g = float(input("Enter a weight in grams: "))


_kg = weight1_kg + weight2_kg

_g = weight1_g + weight2_g

extra_g=_g//1000

if _g>1000:
  _g-=extra_g*1000
  _kg+=extra_g

print(f'{_kg} kg {_g} g')

File: java/class 5/Question8.java
import java.util.Scanner;

public class Question8 {
  public static void main(String[] args) {
    Scanner input = new Scanner(System.in);
    System.out.print("Enter an amount in kilograms: ");
    double weight1kg=input.nextDouble();
    System.out.print("Enter an amount in grams: ");
    double weight1g=input.nextDouble();
    double weight2kg=input.nextDouble();
    System.out.print("Enter an amount in kilograms: ");
    double weight2g=input.nextDouble();
    System.out.print("Enter an amount in grams: ");
    double weightkg = weight1kg + weight2kg;
    double weightg = weight1g + weight2g;
    if (weightg>1000) {
      double extrag = weightg%1000;
      weightkg += extrag;
    }
    System.out.println("The total weight is: " + weightkg + " kg and " + weightg + " g");
    input.close();
  }
}
